### PR TITLE
chore(thanos): new image repository and workflow added

### PR DESCRIPTION
**Pull request by viennaa**
Switching to external image repository and adding to helm build workflow

### CODE REVIEW

**Review by kengou — APPROVED** [review]

**Comment by richardtief** [comment]
Thank you! We also need it here: https://github.com/cloudoperators/greenhouse-extensions/blob/6a05d264dfb84e05aa66b19ee6faf2f557a6c886/.github/workflows/helm-release.yaml#L17-L18 to make the gh action react to changes

File: thanos/plugindefinition.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ spec:
     description: thanos
     helmChart:
         name: thanos
-        repository: "oci://keppel.eu-de-1.cloud.sap/greenhouse-extensions"
-        version: 0.0.8
+        repository: "oci://ghcr.io/cloudoperators/greenhouse-extensions/charts "
```
Comment (ibakshay) [inline thanos/plugindefinition.yaml:12]: There is a trailing blank space. Is this intended? 
```suggestion
        repository: "oci://ghcr.io/cloudoperators/greenhouse-extensions/charts"
```

**Review by ibakshay — APPROVED** [review]
🚀

**Review by richardtief — APPROVED** [review]
Looks good now. Thanks

File: thanos/plugindefinition.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ spec:
     description: thanos
     helmChart:
         name: thanos
-        repository: "oci://keppel.eu-de-1.cloud.sap/greenhouse-extensions"
+        repository: "oci://ghcr.io/cloudoperators/greenhouse-extensions/charts"
         version: 0.0.8
```
Comment (IvoGoman) [inline thanos/plugindefinition.yaml:13]: Can you bump to `0.0.9`? Overlooked it on #297

**Review by IvoGoman — APPROVED** [review]

**Pull request merged**